### ISSUE TITLE
new 0.8 warning in console during combat

### ISSUE THREAD

**Issue by scooper4711** [toastygm/HarnMaster-3-FoundryVTT#273]
During combat, I noticed the following warning you will want to clean up
```
You are passing a Token instance to _getSpeakerFromToken which now expects a TokenDocument instance instead
_getSpeakerFromToken @ foundry.js:12800
getSpeaker @ foundry.js:12748
ignoreResume @ combat.js:1134
ignoreResume @ macros.js:1185
_onChatCardAction @ actor.js:854
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```

**Comment by toastygm** [comment]
This is a foundry bug, not a HM3 bug.

**Comment by toastygm** [comment]
This foundry bug was already reported, and it will be fixed in 0.8.9:

https://gitlab.com/foundrynet/foundryvtt/-/issues/5559
